app/jobs/users/bust_cache_job.rb

```diff
@@ -0,0 +1,15 @@
+module Users
+  class BustCacheJob < ApplicationJob
+    queue_as :users_bust_cache
+
+    def perform(user_id, cache_buster = CacheBuster.new)
+      user = User.find_by(id: user_id)
+
+      cache_buster.bust("/#{user.username}") if user
```

Comment:
no need to check "if user" for each line, you can just return from the method with `return unless user` after line 6 :D Fine work nonetheless!